.github/workflows/build-wheels.yml

```diff
@@ -0,0 +1,31 @@
+name: Build wheels
+
+on:
+  workflow_dispatch
+
+jobs:
+  buildwheels:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
```

Comment:
Are we not building on windows because we haven't setup the necessary machinery yet or?

Reply:
Do you think it'll be useful to open an issue regd the missing windows wheels?

Reply:
Done - https://github.com/enthought/vpsearch/issues/13.